lib/service/Dialog.dart

```diff
@@ -1333,74 +1333,80 @@ Dialog accountDialog(BuildContext context) {
-                              child: Column(
-                                mainAxisAlignment:
-                                    MainAxisAlignment.spaceAround,
-                                // crossAxisAlignment: CrossAxisAlignment.start,
-                                children: <Widget>[
-                                  Text("Connected With Metamask",
+                            height: 65,
+                            child: Column(
+                              mainAxisAlignment: MainAxisAlignment.spaceAround,
+                              // crossAxisAlignment: CrossAxisAlignment.start,
+                              children: <Widget>[
+                                Text("Connected With Metamask",
+                                    style: textStyle(
+                                        Colors.grey[600]!, 13, false)),
+                                Row(
+                                  mainAxisAlignment: MainAxisAlignment.start,
+                                  children: <Widget>[
+                                    const Icon(
+                                      Icons.account_balance_wallet,
+                                      color: Colors.white,
+                                    ),
+                                    Text(
+                                      "$retStr",
+                                      style: textStyle(Colors.white, 20, false),
+                                    ),
+                                  ],
+                                ),
+                              ],
+                            ),
+                          ),
+                          Container(
+                            height: 65,
+                            child: Column(
+                              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                              children: [
+                                // There's only MetaMask currently supported,
+                                // so there's no point in having a change
+                                // wallet button yet.
+
+                                // Container(
+                                //   width: 75,
+                                //   height: 25,
+                                //   decoration: boxDecoration(Colors.transparent,
+                                //       100, 0, Colors.blue[800]!),
+                                //   child: TextButton(
+                                //     onPressed: () {
+                                //       controller.changeAddress();
+                                //     },
+                                //     child: Text(
+                                //       "Change",
+                                //       style: textStyle(
+                                //           Colors.blue[300]!, 10, true),
+                                //     ),
+                                //   ),
+                                // ),
```

Comment:
If this needs to be added in later lets create a ticket for that and tag a TODO here.

Reply:
You're right. Done!

Reply:
Thanks!